guides/source/active_record_validations.md

```diff
@@ -1073,7 +1073,7 @@ Passing an array of symbols is also acceptable.
 
 ```ruby
 class Book
-  include ActiveModel:Validations
+  include ActiveModel::Validations
```

Comment:
Regardless of the style consistency wins, the fact this does a `Ripper.sexp` to check the syntax of the code caught this which is a huge benefit IMO.